Peralta/template/simple_functions.py

```diff
@@ -10,3 +10,19 @@ def factorial(value):
         return 1
     else:
         return value * factorial(value - 1)
+
+def is_prime(value):
+    assert type(value) is int
+    
+    # Initialise the result as True
+    result = True
+
+    # Try to find a integer which divides the input value:
+    # If we find one, the result will be changed to False
+    for i in range(2, value):
```

Comment:
This could be improved by going up to about sqrt(n).

Reply:
I will implement this improvement :-)